Stesura delle linee guida per ciò che concerne le Relations (#13)
Il-Libro-Open-Source/governance: GUIDELINES-RELATIONS.md

# Intento del documento

> Questo documento identifica delle semplici e basilari linee guida quando si intrattengono relazioni a nome del progetto.

Reviewer: Questa frase mi suona complicata, ma non saprei come rielaborarla :rofl:

Author: Non saprei come rigirarla

> Queste linee guida sono da intendersi come regole _di buon senso_ e di corretta comunicazione.

Reviewer: ```suggestion
Queste linee guida sono da intendersi come regole _del buon senso_ e di corretta comunicazione.
```

## Relazioni con altri progetti o community

> Il coinvolgimento delle persone è un aspetto fondamentale che sta alla base di questo progetto, di conseguenza intraprendere relazioni con progetti che sposano la nostra stessa filosofia è un aspetto fondamentale.  

Reviewer: Per evitare la ripetizione di "è un aspetto fondamental"

```suggestion
Il coinvolgimento delle persone è un aspetto fondamentale che sta alla base di questo progetto, di conseguenza è importante relazionarsi con progetti che sposano la nostra stessa filosofia.  
```

Lo stesso vale per le community o per qualsiasi gruppo formatosi attorno a filosofie simili alla nostra.

## Pubblicazione di contenuti

> La pubblicazione di contenuti, che siano essi video, slide, articoli o post sui social, è un aspetto fondamentale per la crescita del progetto, di conseguenza è importante che i contenuti pubblicati siano consoni e che rispettino le linee guida del progetto.

Reviewer: Anche qui, per evitare ripetizioni

```suggestion
La pubblicazione di contenuti, che siano essi video, slide, articoli o post sui social, è imprescindibile per la crescita del progetto, di conseguenza è importante che i contenuti pubblicati siano consoni e che rispettino le linee guida del progetto.
```

> Per quanto possibile, si ricorda di mantenere un linguaggio semplice, gender-neutral e che rispetti il nostro [Codice di Comportamento](https://github.com/Il-Libro-Open-Source/book/blob/main/CODE_OF_CONDUCT.md).  

Author: ```suggestion
Per quanto possibile, si ricorda di mantenere un linguaggio semplice, [gender-neutral](https://github.com/Il-Libro-Open-Source/book/blob/main/GUIDELINES.md#espressioni-di-esempio) e che rispetti il nostro [Codice di Comportamento](https://github.com/Il-Libro-Open-Source/book/blob/main/CODE_OF_CONDUCT.md).  
```

> Chiaramente non è possibile controllare ogni singolo contenuto pubblicato o analizzare in tempo reale ogni frase detta durante una discussione, un meetup o una live, ma è importante che ci si ricordi che quando si parla a nome del progetto ogni parola detta è importante e può avere un impatto positivo o negativo sulla percezione del progetto stesso.

Reviewer: Nit:

```suggestion
Chiaramente non è possibile controllare ogni singolo contenuto pubblicato o analizzare in tempo reale ogni frase detta durante una discussione, un meetup o una live, ma è importante ricordare che quando si parla a nome del progetto ogni parola detta ha un peso e può avere un impatto positivo o negativo sulla percezione del progetto stesso.
```

Reviewer 2: > Nit:

Aggiungerei anche il riferimento alle linee guida accanto a "gender-neutral"

## Hashtag

Non avendo ancora stabilito un titolo ufficiale per il progetto, si consiglia di utilizzare gli hashtag `#IlLibroOpenSource` e `#LibroOpenSource` per identificare il progetto, i suoi contenuti e le sue attività.

## Esposizione dei valori del progetto

Il progetto si basa su alcuni valori fondamentali, che sono:

- `Open Source`: Il progetto vive grazie al supporto di tutti, e tutti possono contribuire in qualsiasi modo ritengano opportuno.

> - `Open Governance`: Il progetto è aperto a tutti, non esistono gerarchie o ruoli, e tutti possono contribuire in qualsiasi modo ritengano opportuno. Chiunque può proporre un cambiamento o una modifica, e tutti possono esprimere la propria opinione.

Reviewer: Mmh tecnicamente alcuni hanno dei ruoli ben precisi (con precise responsabilità), se lasciassimo solo "non esistono gerarchie"?

Author: ```suggestion
- `Open Governance`: Il progetto è aperto a tutti, non esistono gerarchie e tutti possono contribuire in qualsiasi modo ritengano opportuno. Chiunque può proporre un cambiamento o una modifica, e tutti possono esprimere la propria opinione.
```

Author: @AngeloAvv ti torna?

Author: Niente, non avevo visto la proposta sotto.

> - `Open Governance`: Il progetto è aperto a tutti, non esistono gerarchie o ruoli, e tutti possono contribuire in qualsiasi modo ritengano opportuno. Chiunque può proporre un cambiamento o una modifica, e tutti possono esprimere la propria opinione.

Reviewer: Mmh tecnicamente alcuni hanno dei ruoli ben precisi (con precise responsabilità), se lasciassimo solo "non esistono gerarchie"?

Ho proposto dei cambiamenti per evitare ripetizioni con la frase di sopra

```suggestion
- `Open Governance`: Il progetto è aperto a tutti, non esistono gerarchie, chiunque può dare una mano. Ciascuno di noi può proporre un cambiamento o una modifica, e tutti possono esprimere la propria opinione.
```

> - `Collaborazione`: Avere la stessa opinione su tutto, in una qualsiasi community, è piuttosto difficile, se non addirittura impossibile. È importante che tutti possano esprimere la propria opinione e che tutti abbiano come obiettivo comune il miglioramento del progetto e la collaborazione reciproca, a testimonianza della nostra convinzione che insieme possiamo creare qualcosa di più grande della somma delle sue parti.

Reviewer: ```suggestion
- `Collaborazione`: Avere la stessa opinione su tutto, in una qualsiasi community, è piuttosto difficile, se non addirittura impossibile. È importante che tutti possano esprimere la propria opinione e che tutti abbiano come obiettivo comune il miglioramento del progetto e la collaborazione reciproca, a testimonianza della nostra idea che insieme possiamo creare qualcosa di più grande della somma delle sue parti.
```

## Esposizione del concetto di Ambassador

Il concetto di Ambassador è stato introdotto per identificare le persone che si impegnano in prima linea nella stesura dei contenuti o di altri aspetti fondamentali del progetto.

> Nonostante questo, è importante ricordare che il progetto è aperto a tutti e che non esistono gerarchie o ruoli, e che le persone che ricoprono il ruolo di Ambassador sono più dei _garanti_ delle linee guida e dei valori del progetto che dei _leader_ da seguire.

Reviewer 3: @Cadienvan metterei in questo punto anche una descrizione di come si diventa Ambassador, specificando le regole attraverso il sistema di votazione e i momenti in cui si può entrare all'interno dell'organizzazione

> Nonostante questo, è importante ricordare che il progetto è aperto a tutti e che non esistono gerarchie o ruoli, e che le persone che ricoprono il ruolo di Ambassador sono più dei _garanti_ delle linee guida e dei valori del progetto che dei _leader_ da seguire.

Reviewer: ```suggestion
Nonostante questo, è importante ricordare che il progetto è aperto a tutti e che non esistono gerarchie, e che le persone che ricoprono il ruolo di Ambassador sono più dei _garanti_ delle linee guida e dei valori del progetto che dei _leader_ da seguire.
```

## Esposizione del concetto di Working Group

> Il concetto di Working Group è stato introdotto per identificare i gruppi di lavoro che si occupano di aspetti specifici del progetto.

Reviewer 3: Metterei una descrizione più specifica del working group ad esempio:

Siccome gestire un'organizzazione di questa portata può essere dipendioso e complicato, per facilitare la gestione di alcuni aspetti organizzativi sono stati introdotti dei Working Group.

Un Working Group è formato da diversi Ambassador che si occupano di gestire un particolare aspetto legato al progetto. Ogni working group ha anche uno specifico tag su Gitlab.

Essere all'interno di un working group vuol dire esser responsabili direttamente di una serie di task e attività che sono state condivise all'interno del gruppo.

Tutti possono collaborare ad un working group, le call vengono organizzate all'interno del gruppo telegram e le note degli incontri sono pubbliche all'interno del sotto-repo Governance.
L'ultima parola su alcune decisioni importanti tuttavia spetta agli Ambassador eletti e facenti parte di un determinato gruppo.

Il motivo di questa scelta non è per chiudere l'organizzazione, ma per facilitare ed agevolare il processo decisionale che tiene conto delle contribuzioni personali e singole, ma necessita comunque di una revisione organizzativa di almeno 3 membri all'interno di un working group.

> Il concetto di Working Group è stato introdotto per identificare i gruppi di lavoro che si occupano di aspetti specifici del progetto.

Reviewer: ```suggestion
Il concetto di Working Group è stato introdotto per identificare dei gruppi di lavoro che si occupano di aspetti specifici del progetto.
```

> È importante ricordare che non esistono gruppi con più o meno poteri di altri, ma che ci sono dei perimetri ben definiti per ognuno di essi e che fare o meno parte di uno Working Group non è un aspetto che definisce la _posizione_ di una persona all'interno del progetto.

Reviewer: ```suggestion
È importante ricordare che non esistono gruppi con più o meno poteri di altri, ma che ci sono dei perimetri ben definiti per ognuno di essi e che fare o meno parte di uno Working Group non è un aspetto che definisce la _posizione gerarchica_ di una persona all'interno del progetto.
```

## Partecipazione a eventi

> La partecipazione a eventi è un aspetto fondamentale per la crescita del progetto, di conseguenza è importante che le persone che partecipano a eventi a nome del progetto siano in grado di rappresentarlo al meglio.

Reviewer: ```suggestion
La partecipazione a eventi è un aspetto fondamentale per la crescita del progetto, di conseguenza è importante che le persone che vi partecipino a nome del progetto siano in grado di rappresentarlo al meglio.
```

> Non vengono posti limiti o vincoli alla partecipazione a eventi, meetup e similari, ma è importante che vi sia una previa comunicazione e condivisione di eventuali slide o contenuti che verranno utilizzati durante l'evento riguardo al progetto.

Reviewer: ```suggestion
Non vengono posti limiti o vincoli alla partecipazione a eventi, meetup e similari, ma è importante che vi sia una previa comunicazione e condivisione di eventuali slide o contenuti a tema che verranno utilizzati durante l'evento.
```

> Questo non è un modo per limitare la libertà di espressione delle persone, ma anzi è un modo per garantire che i contenuti siano consoni e che rispettino le linee guida del progetto, oltre che essere una fonte di scambio di informazioni e arricchimento reciproco.

Reviewer: ```suggestion
Questo non è un modo per limitare la libertà di espressione delle persone, bensì un modo per garantire che i contenuti siano consoni e che rispettino le linee guida del progetto, oltre che essere una fonte di scambio di informazioni e arricchimento reciproco.
```
